pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiTest.java

```diff
@@ -798,7 +798,6 @@ public void partitionedTopics(String topicName) throws Exception {
 
         try {
             admin.topics().getPartitionedStats(partitionedTopicName, false);
-            fail("should have failed");
```

Comment:
This line deletion makes the test block no mean.  How about add some comments of the expected stats result, and  verify it is output as expected.

Reply:
> This line deletion makes the test block no mean. How about add some comments of the expected stats result, and verify it is output as expected.

ok